homu/main.py

```diff
@@ -286,6 +289,25 @@ def parse_commands(body, username, repo_cfg, state, my_username, db, *, realtime
 
             state.save()
 
+        elif word.startswith('delegate='):
+            if try_only:
+                state.add_comment(':key: Insufficient privileges')
+                continue
+            state.delegate = word[len('delegate='):]
+            state.save()
+
+        elif word == 'delegate-':
+            state.delegate = ''
+            state.save()
+
+        elif word == 'delegate+':
+            if try_only:
+                state.add_comment(':key: Insufficient privileges')
+                continue
+            state.delegate = state.get_repo().pull_request(state.num).user
+            state.add_comment(':v: @{} can now approve this pull request'.format(state.delegate))
```

Comment:
This was basically to mirror how `r+ <commit>` vs `r+` works. The one where something is autoselected is where homu reports _what_ was autoselected. I can make it report in the other one too though. *shrug*